python/ray/autoscaler/commands.py

```diff
@@ -455,7 +455,8 @@ def kill_node(config_file, yes, hard, override_cluster_name):
 
 def monitor_cluster(cluster_config_file, num_lines, override_cluster_name):
     """Tails the autoscaler logs of a Ray cluster."""
-    cmd = "tail -n {} -f /tmp/ray/session_*/logs/monitor*".format(num_lines)
+    cmd = "tail -n {} -f /tmp/ray/session_latest/logs/monitor*".format(
+        num_lines)
```

Comment:
can we f-string this?

Reply:
i just moved the #10702 part to a different pr see #10705